source/howto/AddUsersOpenStackVMInstance.rst

```diff
@@ -0,0 +1,148 @@
+##################################################
+Create additional user(s) on OpenStack VM/Instance
+##################################################
+
+Introduction
+############
+
+This guide explains how to create users in Linux using the command line and the *useradd* command and  grant administrator privileges using *sudo*.
+
+The `sudo` command provides a mechanism for granting administrator privileges — ordinarily only available to the root user — to normal users. 
+
+This guide will show you how to create a new user with `sudo` access on Linux, without having to modify your server’s `/etc/sudoers` file.
+
+Prerequisites
+#############
+
+* Valid Login and SSH or Console Access to VM
+* Have sudo privileges
+* VPN (if outside the network)
+
+Adding Users
+############
+
+#. Logging into you server
+
+    SSH in to your server as using your FEDADMIN user:
+
+
+    .. code-block:: shell
+
+      %ssh FEDID@172.16.113.47
+        ____ _____ _____ ____    ____ _                 _
+      / ___|_   _|  ___/ ___|  / ___| | ___  _   _  __| |
+      \___ \ | | | |_ | |     | |   | |/ _ \| | | |/ _` |
+        ___) || | |  _|| |___  | |___| | (_) | |_| | (_| |
+      |____/ |_| |_|   \____|  \____|_|\___/ \__,_|\__,_|
+
+                Location: r89.harwell.europe hpd r89rack241
+                  Branch: xfu59478/mr_build_centos_aq_image (sandbox)
+              Archetype: cloud
+            Personality: nubesvms
+        Operating System: centos7x-x86_64
+          Snapshot Date: 2019-05-20
+
+#. Login as Root
+
+    Use the `sudo` command to login in as root:
+
+    .. code-block:: shell
+
+      [FEDID@preprod-horizon1 ~]$ sudo su -
+
+      [root@preprod-horizon1 ~]#
+
+#. Adding a New User to the System using the script (addusers.sh) below. Edit the script and save it.
+
+    .. code-block:: shell
+
```

Comment:
Add the following as the first line of your script, so it knows which interpreter to use

```
#!/bin/sh
```
(Assuming it's not Bash only with Bash-isms)